### PR TITLE
[SQL] fix tmPreferences and tests

### PR DESCRIPTION
**Pull request by jrappen**

### CODE REVIEW

File: SQL/Indentation Rules - Comments.tmPreferences [hunk 1]
```diff
@@ -1,13 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-    <key>scope</key>
```
Comment (michaelblyons) [inline SQL/Indentation Rules - Comments.tmPreferences:5]: From tabs to spaces? Or spaces to tabs? (GitHub not very helpful here.)

File: SQL/tests/symbols/syntax_test_symbols_tsql.sql [hunk 2]
```diff
@@ -0,0 +1,66 @@
+-- SYNTAX TEST "Packages/SQL/TSQL.sublime-syntax"
+
+SELECT *
```
Comment (michaelblyons) [inline SQL/tests/symbols/syntax_test_symbols_tsql.sql:3]: Clean out the indentation parts if you want to split this test into two files.

File: SQL/MySQL.sublime-syntax [hunk 3]
```diff
@@ -123,20 +123,27 @@ contexts:
     - meta_prepend: true
     - match: \b(?i:create\s+or\s+replace)\b
       scope: keyword.other.ddl.sql
-      push: [ddl-create-target, create-condition, ddl-target]
```
Comment (michaelblyons) [inline SQL/MySQL.sublime-syntax:126]: I think there's a mild case for `-`-lists, if only to see at a glance how deep the stack becomes. I'm fine with this change, but I'd also be okay with the original.

File: SQL/Indentation Rules - Comments.tmPreferences (hunk 1 above)
Comment (jrappen) [reply SQL/Indentation Rules - Comments.tmPreferences:5]: I changed spaces to tabs to be consistent with the other files.

**Review by keith-hall — APPROVED** [review]

**Comment by keith-hall** [comment]
Thanks! :)

**Pull request merged**